packages/design-tokens/README.md

```diff
@@ -0,0 +1,36 @@
+# Kaizen Tokens
+Design tokens for all platforms.
+
+## About
+Design Tokens are the heart of every Design System. The tokens represented here are platform-agnostic (JSON), as this will help us contribute to & facilitate the maintenance of living style guides. 
+
+In its current state, this package supports Sass and generates three different outputs: color, typography, helpers.
```

Comment:
This needs to be updated to just color. Also, we should mention that we only include option tokens and not decision tokens (in case people see Warning/Instructional in Figma and wonder why they aren't in this package)